README.md

```diff
@@ -54,30 +129,60 @@ const credential = await navigator.credentials.get({
 });
 ```
 
-## Confirming payment
+## Authenticating a payment
 
-Any origin may invoke the [Payment Request API](https://w3c.github.io/payment-request/) to prompt the user to verify a credential created by any other origin. The `PaymentRequest.show()` method must require a user gesture and display user interface with the amount of payment and the hostname of the top-level context where the `PaymentRequest` API was invoked.
+Any origin may invoke the [Payment Request API](https://w3c.github.io/payment-request/) with the `secure-payment-confirmation` payment method to prompt the user to verify a SecurePaymentCredential created by any other origin. The `PaymentRequest.show()` method must require a user gesture. The browser will display a native user interface with the payment amount and the payee origin, which is taken to be the origin of the top-level context where the `PaymentRequest` API was invoked.
+
+Proposed new `secure-payment-confirmation` payment method:
+
+```webidl
+dictionary SecurePaymentConfirmationRequest {
+  SecurePaymentConfirmationAction action;
+  required DOMString instrumentId;
+  required BufferSource networkData;
+  unsigned long timeout;
+  required USVString fallbackUrl;
+};
+
+enum SecurePaymentConfirmationAction {
+  "authenticate",
```

Comment:
Do we expect more actions in the future?

Reply:
Yes. "select-and-authenticate" may be a future action for a combined instrument selection and authenticate flow. See https://docs.google.com/document/d/1n73fdsn8tTDYUGjQPOGRGI_VcTaGNjweedSgcH7P8Hc/edit?ts=5ef93da4&pli=1#heading=h.y7uxuzpcwy0b